mmdet3d/datasets/det3d_dataset.py

```diff
@@ -318,12 +326,34 @@ def prepare_data(self, index: int) -> Optional[dict]:
                 return None
 
         example = self.pipeline(input_dict)
+
         if not self.test_mode and self.filter_empty_gt:
             # after pipeline drop the example with empty annotations
             # return None to random another in `__getitem__`
             if example is None or len(
                     example['data_samples'].gt_instances_3d.labels_3d) == 0:
                 return None
+
+        if self.show_ins_statistics:
+            logger: MMLogger = MMLogger.get_current_instance()
+            ori_num_per_cat = dict()
+            for label in ori_input_dict['ann_info']['gt_labels_3d']:
+                cat_name = self.metainfo['CLASSES'][label]
+                ori_num_per_cat[cat_name] = ori_num_per_cat.get(cat_name,
+                                                                0) + 1
+            new_num_per_cat = dict()
+            for label in example['data_samples'].gt_instances_3d.labels_3d:
+                cat_name = self.metainfo['CLASSES'][label]
+                new_num_per_cat[cat_name] = new_num_per_cat.get(cat_name,
+                                                                0) + 1
+            content_show = ''
+            for cat_name, num in ori_num_per_cat.items():
+                new_num = new_num_per_cat.get(cat_name, 0)
+                content_show += f'\n{cat_name}'.ljust(25) + f'{new_num}/{num}'
+            logger.info(
```

Comment:
I think you can add some simplified prefixes for print info to distinguish dataset info and data info. And maybe you need to modify [dbsampler info](https://github.com/open-mmlab/mmdetection3d/blob/c9541b0db89498fdea5cafd05b7b17f7b625b858/mmdet3d/datasets/pipelines/dbsampler.py#L139) to add a prefix with different with other info to make it more clear.